src/shared/observability/scope.tsx

```diff
@@ -27,14 +21,10 @@ const integrations = getDefaultIntegrations({}).filter((defaultIntegration) => {
   );
 });
 
-const createClient = (
-  executionEnvironment: SentryExecutionEnvironment,
-  extensionId: string,
-) => {
+const createClient = (executionEnvironment: SentryExecutionEnvironment) => {
   return new BrowserClient({
     dsn: process.env.SENTRY_DSN,
     enabled: process.env.SENTRY_ENVIRONMENT === 'production',
-    allowUrls: [`chrome-://${extensionId}`],
```

Comment:
I removed it because:
1. it was not working - it needs InboundFilter Sentry integration but when dealing with chrome extensions some of the integrations are not working or even should be disabled (sentry docs)
2. errors caught in react-lifecycle are not coming from `chrome-...` but from normal url's like `twitter.com`